Sources: Gentoo
Upstream: No (mostly dead)
Reason: Needed to compile properly

diff --git a/lib/Makefile.am b/lib/Makefile.am
index 3b2556b..b098caa 100644
--- a/lib/Makefile.am
+++ b/lib/Makefile.am
@@ -45,6 +45,7 @@ sasl_version = 2:22:0

 INCLUDES=-I$(top_srcdir)/include -I$(top_srcdir)/plugins -I$(top_builddir)/include -I$(top_srcdir)/sasldb

+AM_CFLAGS = -fPIC
 EXTRA_DIST = windlopen.c staticopen.h NTMakefile
 EXTRA_LIBRARIES = libsasl2.a
 noinst_LIBRARIES = @SASL_STATIC_LIBS@
diff --git a/plugins/Makefile.am b/plugins/Makefile.am
index 3036b91..769e418 100644
--- a/plugins/Makefile.am
+++ b/plugins/Makefile.am
@@ -63,6 +63,7 @@ srp_version = 2:22:0

 INCLUDES=-I$(top_srcdir)/include -I$(top_srcdir)/lib -I$(top_srcdir)/sasldb -I$(top_builddir)/include
 AM_LDFLAGS = -module -export-dynamic -rpath $(plugindir)
+AM_CFLAGS = -fPIC

 COMPAT_OBJS = @LTGETADDRINFOOBJS@ @LTGETNAMEINFOOBJS@ @LTSNPRINTFOBJS@

diff --git a/sasldb/Makefile.am b/sasldb/Makefile.am
index 4b2c593..9177054 100644
--- a/sasldb/Makefile.am
+++ b/sasldb/Makefile.am
@@ -48,6 +48,7 @@ INCLUDES=-I$(top_srcdir)/include -I$(top_builddir)/include @SASL_DB_INC@

 extra_common_sources = db_none.c db_ndbm.c db_gdbm.c db_berkeley.c

+AM_CFLAGS = -fPIC
 EXTRA_DIST = NTMakefile

 noinst_LTLIBRARIES = libsasldb.la
